typescript/infra/src/utils/gcloud.ts

```diff
@@ -182,5 +201,7 @@ function iamConditionsEqual(
   if (a === undefined && b === undefined) {
     return true;
   }
-  return a && b && a.title === b.title && a.expression === b.expression;
+  const isEqual =
```

Comment:
¿por que?

Reply:
linting

I have another task in my mental backlog to update repo settings to auto-lint/format on a file save in vscode

Reply:
ah wait this one is probably from adding a log and then removing the log because it's too verbose